components/Card.tsx

```diff
@@ -4,34 +4,50 @@ import { CldImage } from "next-cloudinary";
 type CardProps = {
   name: string;
   cloudinarySrc: string;
+  languages: string[];
+  gender: string;
+  skill: string;
 };
 
-export default function Card({ name, cloudinarySrc }: CardProps): JSX.Element {
+export default function Card({
+  name,
+  cloudinarySrc,
+  languages,
+  gender,
+  skill,
+}: CardProps): JSX.Element {
   return (
     <StyledDiv>
       <CldImage
         width="80"
         height="80"
         src={cloudinarySrc}
-        alt="a profile image"
+        alt={`the profile image of ${name}`}
```

Comment:
better not to mention picture or image in the `alt` tag, since the screen reader announces "image". Just write the name of the person, that is perfectly fine.

Reply:
ok, nice!